packages/python/plotly/plotly/tests/test_core/test_px/test_px.py

```diff
@@ -182,3 +182,43 @@ def test_px_templates():
     assert fig.layout.xaxis3.showgrid is None
     assert fig.layout.yaxis2.showgrid
     assert fig.layout.yaxis3.showgrid
+
+
+def test_orthogonal_orderings():
```

Comment:
Is it worth testing that adding in `category_orders` a value not present in the data does not break things?

Reply:
yeah I tried but it's a harder test to write and much slower :)

Reply:
OK I think I have a way to spot-check a few options